src/components/home.jsx

```diff
@@ -0,0 +1,11 @@
+import './home.css'
+
+const Home = ({title, author}) => (
```

Comment:
- [ ] [OPTIONAL] Kindly notice that it should be better to name this component `Book` as this will make its name related to its job.

Reply:
Okay, I will do that now